tbump/git_bumper.py

```diff
@@ -145,8 +145,8 @@ def get_commands(self, new_version: str) -> List[Command]:
             self.add_command(
                 res, "tag", "--annotate", "--message", tag_message, tag_name
             )
-        if "push" in self.operations:
+        if "push_commit" in self.operations:
             self.add_command(res, "push", self.remote_name, self.remote_branch)
-            if "tag" in self.operations:
+            if "tag" in self.operations and "push_tag" in self.operations:
```

Comment:
I wouldn't put this logic here. I'd just test for push_tag here because it's what you're doing. Then when parsing the arguments, if there is --no-tag, also remove "push_tag" from the operations.

Reply:
Good point, thanks